src/coreclr/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.CoreCLR.cs

```diff
@@ -236,9 +236,9 @@ private static object PtrToStructureHelper(IntPtr ptr, Type structureType)
         [MethodImpl(MethodImplOptions.InternalCall)]
         internal static extern bool IsPinnable(object? obj);
 
-        internal static bool IsComSupported { get; } = InitializeIsComSupported();
+        internal static bool IsBuiltInComSupported { get; } = InitializeIsBuiltInComSupported();
 
-        private static bool InitializeIsComSupported() => IsComSupportedInternal();
+        private static bool InitializeIsBuiltInComSupported() => IsBuiltInComSupportedInternal();
```

Comment:
What is the value of this method? Why not just call the QCall directly above in `internal static bool IsBuiltInComSupported { get; } = ....`?

Reply:
Thanks!